### ISSUE TITLE
Uninitialized string offset error if sftp is used without privkey

### ISSUE THREAD

**Issue by raulneis** [banago/PHPloy#275]
When sftp schema is used, if no privkey is specified it throws a PHP Notice:

```
PHP Notice:  Uninitialized string offset: 0 in phar:///usr/local/bin/phploy/src/Connection.php on line 104
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/phploy:0
PHP   2. include() /usr/local/bin/phploy:10
PHP   3. Banago\PHPloy\PHPloy->__construct() phar:///usr/local/bin/phploy/phploy.php:11
PHP   4. Banago\PHPloy\PHPloy->deploy() phar:///usr/local/bin/phploy/src/PHPloy.php:273
PHP   5. Banago\PHPloy\Connection->__construct() phar:///usr/local/bin/phploy/src/PHPloy.php:620
PHP   6. Banago\PHPloy\Connection->connectToSftp() phar:///usr/local/bin/phploy/src/Connection.php:37
```

**Comment by Lewiatan** [comment]
same here

**Comment by jjeaton** [comment]
This was fixed in #281
